lib/capdrupal.rb

```diff
@@ -0,0 +1,310 @@
+namespace :load do
+  task :defaults do
+    set :install_composer, true
+    set :install_drush, true
+    set :app_path, 'web'
+    set :config_path, 'config/sync'
+    set :backup_path, 'backups'
+    set :keep_backups, 5
+    set :enable_modules, []
+    set :disable_modules, []
+  end
+end
+
+namespace :drush do
+  desc "Install Drush - only once for bootstraping"
+  task :install do
+    on roles(:app) do
+      within shared_path do
+        execute :composer, 'require drush/drush'
+      end
+    end
+  end
+end
+
+namespace :drupal do
+
+  desc 'Bootstrap Drupal site with drush site-install command'
+  task :bootstrap do
```

Comment:
I propose to extract this task from this gem and put it into our utility gem

Reply:
I agree we should create `drupal:permissions:cleanup`

Reply:
Iagree for a `drupal:bootstrap` task